sov-modules/sov-modules-impl/bank/src/query.rs

```diff
@@ -23,7 +24,9 @@ pub struct TotalSupplyResponse {
     pub amount: Option<Amount>,
 }
 
+#[rpc_gen(client, server, namespace = "bank")]
 impl<C: sov_modules_api::Context> Bank<C> {
+    #[rpc_method(name = "balanceOf")]
```

Comment:
Is it possible to unit test the rpc call?

Reply:
Yes, it should be. I'll add a test

Reply:
This actually proved to be a bit tricky because there's no way to discover the address of a created token (#189). Opened #190 instead